modules/nf-core/ampcombi2/parsetables/main.nf

```diff
@@ -1,47 +1,53 @@
+    val opt_amp_db
+    path opt_amp_db_dir
+    path opt_interproscan
 
     output:
-    tuple val(meta), path("${meta.id}/")                              , emit: sample_dir
-    tuple val(meta), path("${meta.id}/contig_gbks/")                  , emit: contig_gbks
-    tuple val(meta), path("${meta.id}/${meta.id}_diamond_matches.txt"), emit: txt
-    tuple val(meta), path("${meta.id}/${meta.id}_ampcombi.tsv")       , emit: tsv
-    tuple val(meta), path("${meta.id}/${meta.id}_amp.faa")            , emit: faa
-    tuple val(meta), path("${meta.id}/${meta.id}_ampcombi.log")       , emit: sample_log, optional:true
-    tuple val(meta), path("Ampcombi_parse_tables.log")                , emit: full_log, optional:true
-    tuple val(meta), path("amp_ref_database/")                        , emit: results_db, optional:true
-    tuple val(meta), path("amp_ref_database/*.dmnd")                  , emit: results_db_dmnd, optional:true
-    tuple val(meta), path("amp_ref_database/*.clean.fasta")           , emit: results_db_fasta, optional:true
-    tuple val(meta), path("amp_ref_database/*.tsv")                   , emit: results_db_tsv, optional:true
-    path "versions.yml"                                               , emit: versions
+    tuple val(meta), path("${meta.id}/")                                , emit: sample_dir
+    tuple val(meta), path("${meta.id}/contig_gbks/")                    , emit: contig_gbks
+    tuple val(meta), path("${meta.id}/${meta.id}_mmseqs_matches.tsv")   , emit: db_tsv
+    tuple val(meta), path("${meta.id}/${meta.id}_ampcombi.tsv")         , emit: tsv
+    tuple val(meta), path("${meta.id}/${meta.id}_amp.faa")              , emit: faa
+    tuple val(meta), path("${meta.id}/${meta.id}_ampcombi.log")         , emit: sample_log   , optional:true
+    tuple val(meta), path("Ampcombi_parse_tables.log")                  , emit: full_log     , optional:true
+    tuple val(meta), path("amp_${opt_amp_db}_database/")                , emit: db           , optional:true
+    tuple val(meta), path("amp_${opt_amp_db}_database/*.txt")           , emit: db_txt       , optional:true
+    tuple val(meta), path("amp_${opt_amp_db}_database/*.fasta")         , emit: db_fasta     , optional:true
+    tuple val(meta), path("amp_${opt_amp_db}_database/mmseqs2/")        , emit: db_mmseqs    , optional:true
+    path "versions.yml"                                                 , emit: versions
 
     when:
     task.ext.when == null || task.ext.when
 
     script:
     def args = task.ext.args ?: ''
     def prefix = task.ext.prefix ?: "${meta.id}"
-    def db = opt_amp_db? "--amp_database $opt_amp_db": ""
+    def db_dir = opt_amp_db_dir ? "--amp_database_dir ${opt_amp_db_dir}" : ""
+    def interpro = opt_interproscan ? "--interproscan_output ${opt_interproscan}" : ""
+
     """
     ampcombi parse_tables \\
-    --path_list '${amp_input.collect{"$it"}.join("' '")}' \\
+    --path_list '${amp_input.collect { "${it}" }.join("' '")}' \\
```

Comment:
I don' think you need the `.collect` here, it should already a list when it comes into the module.

```suggestion
    --path_list '${amp_input.join(" ")}' \\
```

I also assume you just need a space separated input?

Reply:
from what i remember Louisa and I tried it at the beginning of setting up this module and it only worked with `collect` for some reason.